BusyList/Parsing/CommandGrammar.cs

```diff
@@ -49,10 +52,21 @@ public static class CommandGrammar
             from keyword in _keywordDone
             select new DoneCommand(id);
 
+        private static readonly Parser<Command> _editCommand =
+            from id in _number
+            from _ in Parse.WhiteSpace
+            from keyword in _keywordEdit
+            from __ in Parse.WhiteSpace
+            from property in Parse.LetterOrDigit.AtLeastOnce().Text()
```

Comment:
Here, as an example, `4 edit Description bla bla` works, but `4 edit description bla bla` doesn't , so I think  `[property]` should be case insensitive